apps/integration/src/app/home/home.component.html

```diff
@@ -4,4 +4,9 @@
       >Vertical Tabset with Back To Top</a
     >
   </li>
+  <li>
```

Comment:
Nitpicky - could you potentially place this new one first so that they are in alphabetical order? That may make it easier to find things quickly once we add more things.